config/settings.yml

```diff
@@ -464,6 +464,16 @@ vba_documents:
     topic_arns:
       - vetsgov-arn
       - vagov-arn
+  slack:
+    enabled: false
+    notification_url:
+    internal_notification_url:
```

Comment:
See the naming comment below.  I don't have a handle on the credstash changes yet, but I assume changing line 468 is doable too.  Is it?